src/BPEssentials/CooldownHandler.cs

```diff
@@ -101,5 +100,28 @@ public int GetCooldown(string ID, string type, string key)
             if (!Cooldowns.ContainsKey(ID) || !Cooldowns[ID].ContainsKey(type) || !Cooldowns[ID][type].ContainsKey(key)) { return 0; }
             return Cooldowns[ID][type][key];
         }
+
+        private IEnumerator MethodTimer(int time, Action action, int repeat = -1)
+        {
+            int i = 0;
+            Core.Instance.Logger.Log("Starting cool down loop");
+            while (i < repeat && repeat != -1)
```

Comment:
```suggestion
            while (i <= repeat && repeat != -1)
```
That = belongs there tho doesnt it?

Reply:
no it does not, think about it. let say repeat = 1 that would mean the i = 0, meaning the loop would start once then stop when i == 1